vaapi/vaapidisplay_unittest.cpp

```diff
@@ -101,19 +101,6 @@ class VaapiDisplayTest : public ::testing::Test {
 #define VAAPI_DISPLAY_TEST(name) \
     TEST_F(VaapiDisplayTest, name)
 
-#if __ENABLE_X11__
-VAAPI_DISPLAY_TEST(CreateInvalidHandleX11) {
-    FAIL(); // remove when fixed;
-
-    // FIXME: This causes segmentation fault.
-    // VaapiDisplay should be able to handle this gracefully
-    NativeDisplay native = {1, NATIVE_DISPLAY_X11};
-    DisplayPtr display = VaapiDisplay::create(native);
-
-    EXPECT_FALSE(display.get());
-}
-#endif
-
```

Comment:
I think Death Tests used for internal state checking.  In our case, we will crash in vaGetDisplay. It should not apply to death test, right?